lib/coqProject_file.ml4

```diff
@@ -61,6 +63,7 @@ let mk_project project_file makefile install_kind use_ocamlopt = {
   q_includes = [];
   extra_args = [];
   defs = [];
+  cmdline_vfiles = [];
```

Comment:
I think it would be more idiomatic to change the type of `v_files` to `string * [ `CmdLine | `ProjFile ] and adapt the rest of the code.

Reply:
>I think it would be more idiomatic to change the type of `v_files` to ``string * [ `CmdLine | `ProjFile ]`` and adapt the rest of the code. 

I looked into it and you end up having to `map fst` a bunch of times (eg `(not_tops v_files &&
      not_tops mli_files && not_tops ml4_files && not_tops ml_files &&
      not_tops mllib_files && not_tops mlpack_files)`). Maybe if we attach the source information to every field as then it could be treated uniformly.

WDYT?

Reply:
Yep, sounds good.
Maybe make a record `'a thing = { origin : [ 'CmdLine | 'ProjFile ] ; value : 'a }`

Reply:
> Maybe make a record 'a thing = { origin : [ 'CmdLine | 'ProjFile ] ; value : 'a }

Please have a look at the new commit.